README.md

```diff
@@ -222,14 +216,14 @@ JavaScript-centered trainings.
 
 ## Browser support
 
-Volto works well with any modern (and updated) browser, including their mobile
+Volto works well with any modern (evergreen) browser, including their mobile
```

Comment:
What does "evergreen" mean? I've never heard that term used for a web browser, but only for trees and shrubbery.

Reply:
It seems it's a trendy word now... I've already seen it in a number of places refers to those browsers that are automatically updated... so all modern browsers.